genesyscloud/routing_email_route/resource_genesyscloud_routing_email_route_schema.go

```diff
@@ -160,8 +166,10 @@ func RoutingEmailRouteExporter() *resourceExporter.ResourceExporter {
 			"reply_email_address.route_id":  {RefType: "genesyscloud_routing_email_route"},
 		},
 		RemoveIfMissing: map[string][]string{
-			"reply_email_address": {"route_id"},
+			"reply_email_address": {"route_id", "self_reference_route"},
+		},
+		CustomAttributeResolver: map[string]*resourceExporter.RefAttrCustomResolver{
+			"reply_email_address.self_reference_route": {ResolverFunc: resourceExporter.ReplyEmailAddressSelfReferenceRouteExporterResolver},
 		},
-		AllowZeroValues: []string{"from_email"},
```

Comment:
I deleted this because the API never really supports setting a blank from_email field. Always requires at least one or more characters.